includes/cli/class-amp-cli-validation-command.php

```diff
@@ -248,9 +240,11 @@ private function get_validation_url_provider() {
 		}
 
 		$this->scannable_url_provider = new ScannableURLProvider(
-			$limit_type_validate_count,
-			$include_conditionals,
-			$force_crawl_urls
+			new URLScanningContext(
```

Comment:
This can be addressed later, but shouldn't both `ScannableURLProvider` and `URLScanningContext` be instantiated via the injector?

Reply:
Yes, I have refrained so far from moving this class into the DI architecture, but it could be done easily.